### PR TITLE
Add mock data for preview and make use of the visualisation enum

### LINKED ISSUE #109: Show correct visualisation on data source selection page
### Description
Right now it tells you which visualisation it should display. You can likely copy a lot of the logic from visualisation editor page.

### Subtasks
- [x] Display correct visualisation

### PR DESCRIPTION
**Pull request by Kpaubert**
Add charts with mocked data in the data source selection page.
Make use of the VisualisationType enum instead of just string matching.

Closes #109

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import VisualisationEditor from './components/organisms/VisualisationEditor';
 import NotFound from './components/organisms/NotFound';
 
 const client = new ApolloClient({
-    uri: process.env.REACT_APP_GRAPHQL_ENDPOINT,
+    uri: process.env.REACT_APP_GRAPHQL_ENDPOINT, // `https://sintef01.azurewebsites.net/graphql`, //
```
Comment (FredrikAugust) [inline src/App.tsx]: Remove comment

File: src/stories/DataResultItem.stories.tsx [hunk 2]
```diff
@@ -12,11 +13,11 @@ const Template: Story<DataResultItemProps> = (args) => <DataResultItem {...args}
 
 export const Primary = Template.bind({});
 Primary.args = {
-    title: 'This is an title',
+    title: 'This is a title',
     description:
         'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
     tags: ['Energi', 'Miljø', 'Helse', 'Pils', 'Raketter'],
-    visualisationType: 'thresholdchart',
+    visualisationType: VisualisationType.THRESHOLD, // 'thresholdchart',
```
Comment (FredrikAugust) [inline src/stories/DataResultItem.stories.tsx]: Remove comment

File: src/stories/DataResultItem.stories.tsx [hunk 3]
```diff
@@ -12,11 +13,11 @@ const Template: Story<DataResultItemProps> = (args) => <DataResultItem {...args}
 
 export const Primary = Template.bind({});
 Primary.args = {
-    title: 'This is an title',
+    title: 'This is a title',
```
Comment (FredrikAugust) [inline src/stories/DataResultItem.stories.tsx:16]: absolutely right, sir

File: src/components/molecules/DataResultItem.tsx [hunk 4]
```diff
@@ -26,8 +33,37 @@ const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tag
...
+                                        thresholdValue={undefined}
+                                        yLabel="Threshold chart"
+                                    />
+                                );
+                            default:
+                                return <Text>{visualisationType}</Text>;
+                        }
+                    }}
+                </ParentSize>
+                {/*<Text>{visualisationType}</Text>*/}
```
Comment (FredrikAugust) [inline src/components/molecules/DataResultItem.tsx]: Remove comment

File: src/components/molecules/DataResultItem.tsx [hunk 5]
```diff
@@ -26,8 +33,37 @@ const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tag
...
+                                        colorBottom="#E0EEEE"
+                                    />
+                                );
+                            case VisualisationType.THRESHOLD:
+                                return (
+                                    <ThresholdChart
+                                        width={parent.width}
+                                        height={height > 0 ? height : parent.height}
+                                        data={timeEntryMockData}
+                                        thresholdValue={undefined}
```
Comment (FredrikAugust) [inline src/components/molecules/DataResultItem.tsx]:
```suggestion
```

File: src/components/molecules/DataResultItem.tsx [hunk 6]
```diff
@@ -1,15 +1,22 @@
...
 export type DataResultItemProps = {
-    visualisationType: string;
+    visualisationType: VisualisationType;
     title: string;
     description: string;
     tags: string[];
 };
 
 const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tags, visualisationType }) => {
+    const timeEntryMockData = mockTimeEntry(100, undefined);
```
Comment (FredrikAugust) [inline src/components/molecules/DataResultItem.tsx]: What is the second param?

**Review by FredrikAugust — CHANGES_REQUESTED** [review]

File: src/components/molecules/DataResultItem.tsx (hunk 6 above)
Comment (Kpaubert) [reply src/components/molecules/DataResultItem.tsx]: Minmax-tuple, defaults to [0, 100]. Could be changed, but felt it was decent as is.

Comment (FredrikAugust) [reply src/components/molecules/DataResultItem.tsx]: If it defaults, you could leave out the `undefined`.

Comment (Kpaubert) [reply src/components/molecules/DataResultItem.tsx]: That is very very true.

**Review by FredrikAugust — APPROVED** [review]

**Pull request merged**